pkgs/by-name/cs/csvquote/package.nix

```diff
@@ -35,7 +39,7 @@ stdenv.mkDerivation rec {
 
   postInstall = ''
     substituteAllInPlace $out/bin/csvheader
-    patsh $out/bin/csvheader -fs ${builtins.storeDir}
+    patsh $out/bin/csvheader -fs ${builtins.storeDir} --path "$HOST_PATH"
```

Comment:
Can't we wrap the `patsh` binary instead, so instead of changing every derivation we change just one? Also it would be way less error prone in future.

Reply:
for now i fixed every instance that would show up in a ripgrep, wrapping the binary would affect usage outside of nixpkgs derivations as well, so i think a setupHook or a `patshFixupHook` package would be the way to go, but that is a separate PR